syntax = "proto3";

package NDTS.NTestingProcessor;

message TBrockerClientConfig {
    string serverURL = 1;
    string queueName = 2;
}
